bridge/test/LibUtils_test.js

```diff
@@ -97,4 +97,18 @@ contract('LibUtils', async function (accounts) {
         });
     });
 
+    describe('toUint128', async function() {
+        it('Should convert bytes to uint128', async function() {
+            const bytes = accounts[1];
+
+            await truffleAssertions.passes(this.utilsLib.toUint128(bytes, 0));
```

Comment:
You should set a number as byte array, then convert it to uint, then check that the number is the same that you converted.
Just check that it passes is not a good test